api/apps/articles/model/index.coffee

```diff
@@ -48,16 +48,19 @@ Q = require 'bluebird-q'
         debug err if err
         generateArtworks input, article, (err, article) ->
           debug err if err
-          mergeArticleAndAuthor input, article, accessToken, (err, article, author, publishing) ->
-            return callback(err) if err
-            # Merge fullscreen title with main article title
-            article.title = article.hero_section.title if article.hero_section?.type is 'fullscreen'
-            if publishing
-              onPublish article, author, sanitizeAndSave(callback)
-            else if not publishing and not article.slugs?.length > 0
-              generateSlugs article, author, sanitizeAndSave(callback)
-            else
-              sanitizeAndSave(callback)(null, article)
+          publishing = (input.published and not article.published) or (input.scheduled_publish_at and not article.published)
+          article = _.extend article, _.omit(input, 'sections'), updated_at: new Date
+          if input.sections and input.sections.length is 0
+            article.sections = []
```

Comment:
Could we just use `.default([])` in [the schema](https://github.com/artsy/positron/blob/master/api/apps/articles/model/schema.coffee#L108) instead? Seems like that would allow you to do `article = _.extend article, input, updated_at: new Date` without the `omit`